manifests/ssltools/certutil.pp

```diff
@@ -1,5 +1,11 @@
 # type to append cert to nssdb
-define certs::ssltools::certutil($nss_db_dir, $client_cert, $cert_name=$title, $refreshonly = true, $trustargs = ',,') {
+define certs::ssltools::certutil(
+  Stdlib::Absolutepath $nss_db_dir,
+  Stdlib::Absoluatepath $client_cert,
+  String $cert_name = $title,
+  Boolean $refreshonly = true,
+  Boolean $trustargs = ',,'
```

Comment:
I wonder why lint doesn't catch this. Perhaps the trailing commas are not checked on defines.
```suggestion
  Boolean $trustargs = ',,',
```